src/models.py

```diff
@@ -143,3 +140,42 @@ def as_dict(self) -> dict[str, Union[str, dict[str, Union[str, int]]]]:
             mentions=mentions,
             media_house=self.media_house.value,
         )
+
+
+class ExtractorResult(BASE):
+    __tablename__ = "mentions"
+    id = Column(Text, primary_key=True)
+    comment_id = Column(Text, ForeignKey("comments.id", ondelete="CASCADE"))
```

Comment:
I did not understand "ForeignKey("comments.id", ondelete="CASCADE")" does. Maybe you can explain me it later.